### ISSUE TITLE
[Feature Request]: Add feature to temporarily block access to broken tools

### ISSUE THREAD

**Issue by GoldenBanksia** [Hardhat-Enterprises/Deakin-Detonator-Toolkit#989]
### Name of Feature

Tools page update

### Priority

Medium

### Categories

UI, UX, Integration

### Description of Feature

Add a switch, or method of changing the blue ‘Go’ button to ‘In Development’ – rendering the button either not clickable, or greyed out, to remove user access to broken tools. 

This could potentially reduce user frustration, improve user experience, flag broken tools for the development/testing team, and offer a way to filter out the tools that need further work.

Possible extensions: 
1)	Integration with the testing team, where failure of the tool in automated pipeline testing can ‘switch’ the mode back to ‘in development’ (flags for developers, and also removes user frustration)
2)	In development tools could be hidden from view, with a possible toggle button to show/hide them for the user

**Comment by GoldenBanksia** [comment]
Pending review for T1-2025 - will need to review as an ADR

**Comment by TanvirRahmanTaiyeb** [comment]
Closing this issue for now as the corresponding ADR-0022 proposal has been reviewed and rejected due to misalignment with the current project goals. The idea is valuable and may be revisited in the future, but we won’t be proceeding with implementation at this stage. The ADR will remain in the register under "Rejected" to acknowledge the effort and allow others to revisit later if needed.
